.github/workflows/pr.yaml

```diff
@@ -84,12 +85,21 @@ jobs:
         done
       if: steps.list-changed.outputs.changed
 
-    - name: Run chart-testing (install)
+    - uses: actions/setup-go@v4
+      with:
+        go-version: '1.20'
+        cache-dependency-path: "charts/**/ginkgo/go.sum"
+
+    - name: Run ginkgo tests
       run: |
-        ct install \
```

Comment:
Personally, I would not remove this completely but just for the node helm-chart. This is quite useful for having basic chart tests on the other helm-charts.

Reply:
My bad, I removed it for testing and forgot to rollback the change. I reverted it excluding `node` chart as it is not covered by proper tests.